extend.php

```diff
@@ -49,6 +51,9 @@
     (new Extend\ApiSerializer(ForumSerializer::class))
         ->attributes(TrelloAttributes::class),
 
+    (new Extend\Settings())
+        ->serializeToForum('labelTagMappings', 'blomstra-trello.label-tag-mappings'),
```

Comment:
Let's only serialize this if the user has Trello permissions, else we are filling up the payload with unnecessary data